miden-lib/src/accounts/wallets/mod.rs

```diff
@@ -25,9 +26,20 @@ use crate::accounts::{auth::RpoFalcon512, components::basic_wallet_library};
 /// This component supports all account types.
 pub struct BasicWallet;
 
+impl BasicWallet {
+    /// Returns a reference to the Basic Wallet library whose procedures can be imported from
+    /// `account_components::basic_wallet`.
+    ///
+    /// This can be used in the assembly of programs that want to call procedures from this
+    /// component.
+    pub fn library() -> &'static Library {
+        basic_wallet_library()
+    }
```

Comment:
Library is a pretty light-weight object - so, we might as well return clone and `Library` from here.